app/src/main/java/com/hyperether/getgoing/ui/fragment/ShowDataFragment.java

```diff
@@ -117,27 +116,30 @@ public void onDestroy() {
 
     private void initializeViewModel() {
         routeViewModel = new ViewModelProvider(this).get(RouteViewModel.class);
-        routeViewModel.getRouteList().observe(getActivity(), routeList -> {
-            routes.clear();
-            if (routeList.size() > 1) {
-                routeList.remove(0); // remove 0th node
-                for (DbRoute route : routeList) {
-                    if (route.getActivity_id() == activityId) {
-                        routes.add(route);
+        routeViewModel.getAllRoutes().observe(getViewLifecycleOwner(), new Observer<List<DbRoute>>() {
+            @Override
+            public void onChanged(List<DbRoute> dbRoutes) {
+                routes.clear();
+                if (dbRoutes.size() > 1) {
+                    dbRoutes.remove(0); // remove 0th node
```

Comment:
@zukovicfilip why removing 0th node? What data is in 0th node?

Reply:
In GetGoingFragment, node zero is added when app is started for the first time. It was not my code so i left it there.